eslint.js

```diff
@@ -5,21 +5,22 @@ const TaskKitTask = require('taskkit-task');
 const path = require('path');
 
 class EslintTask extends TaskKitTask {
-
   get defaultOptions() {
     return {
-      ignore: ['.git', 'node_modules']
+      ignore: ['.git', 'node_modules'],
+      crashOnError: false
     };
   }
   // returns the module to load when running in a separate process:
   get classModule() {
     return path.join(__dirname, 'eslint.js');
   }
+
   get description() {
     return 'Runs the indicated eslint config against the files you listed, and reports the results ';
   }
 
-  execute(done) {
+  process(done) {
```

Comment:
Why did this change?  Also, we should switch to async since we are making changes

Reply:
Because upgrading of taskkit, there are no async things AFAIK.